### ISSUE TITLE
Does the pointing stick work properly?

### ISSUE THREAD

**Issue by bommbo** [strobo5/T61_PiPico_Scanner#1]
Bro, Does the pointing stick work properly?

**Comment by sly547** [comment]
Yeah, I use it every day and I love it. Just sometimes (every few weeks or twice a day) it stops working and I have to plug the cable in and out. Not often enough to bother me.

This is with the first version. In the second version (tagged 2.0; the one that until now I had failed to `git push` properly) has a reset IC for the Trackpoint, which might help.

As of now the only user of version 2 is my colleague and his X220 keyboard. Here the trackpoint doesn't work properly; it might need its own 3.3V via pin 35 of the connector. Also the Fn key is broken.

**Comment by bommbo** [comment]
Amazing!!!

**Comment by gitaen** [comment]
> Yeah, I use it every day and I love it. Just sometimes (every few weeks or twice a day) it stops working and I have to plug the cable in and out. Not often enough to bother me.
> 
> This is with the first version. In the second version (tagged 2.0; the one that until now I had failed to `git push` properly) has a reset IC for the Trackpoint, which might help.
> 
> As of now the only user of version 2 is my colleague and his X220 keyboard. Here the trackpoint doesn't work properly; it might need its own 3.3V via pin 35 of the connector. Also the Fn key is broken.

Regarding the Fn key not working on the X220 keyboard it is probably due to this same issue https://github.com/thedalles77/USB_Laptop_Keyboard_Controller/issues/22#issue-2895649409.
 
HOTKEY_RTN is located on pin 34 on the T420/X220 keyboard.
